apps/docs/components/index.tsx

```diff
@@ -4,7 +4,7 @@
 
 // Basic UI things
 import Link from 'next/link'
-import { Accordion, Admonition, Alert, Button, CodeBlock, markdownComponents } from 'ui'
+import { Accordion, Admonition, Alert, Button, CodeBlock, cn, markdownComponents } from 'ui'
```

Comment:
Did we need cn? Not seeing it added but I could be overlooking it

Reply:
Oops, just a leftover